ontology/yaml/resources/LIGHTING/entity_types/ABSTRACT.yaml

```diff
@@ -138,3 +138,25 @@ LC:
   uses:
   - illuminance_setpoint
   - illuminance_sensor
+
+DFRT:
+  description: "Lighting Control dimmer fade rate & time"
+  is_abstract: true
+  uses:
+  - dimmer_fade_rate_setpoint
+  - dimmer_fade_time_setpoint
+
+ZLC:
```

Comment:
I take issue with most of these names. Isnt `lighting` redundant since it is associated with a lighting system? Secondly, what devices do you see this being applied to? Gateways? Luminaires?

Additionally, what is a lighting switch? A wall switch? If so, why does run_command not work?

Please explain.

Reply:
ah yes, totally forgot about this. this was from older pull request #386, but we will not be including Ligthing data for now. I am removing all Lighting changes.  

I was still learning DBO when I had this (oct '21) and my first impression was to describe the field accurately, i.e., zone_lighting_switch_command for describing turning on/off lights in an area (zone). It was fairly recent that I learned about run_command/status for the same purpose. You would not believe, I had previously used "power_command" (or something to that effect) until you pointed out power is not the correct one for this hehe.  

much thanks sir.